env/staging/fluentbit.yaml

```diff
@@ -132,22 +137,24 @@ data:
         Use_Kubelet         On
         Kubelet_Port        10250
         Buffer_Size         0
-        Log_Level           info
+        Log_Level           debug
 
     [OUTPUT]
         Name                cloudwatch
+        Alias               notify-app-logs-cloudwatch-output
         Match               application.*
         region              ${AWS_REGION}
         log_group_name      /aws/containerinsights/${CLUSTER_NAME}/application
         log_stream_name     $(tag[4])
         auto_create_group   true
         extra_user_agent    container-insights
-        Log_Level           info
+        Log_Level           debug
```

Comment:
I don't think we want debug set to go as a default and in production! Can we revert these?